_data/glossary/service-account.yaml

```diff
@@ -0,0 +1,10 @@
+id: service-account
+name: Service Account
+full-link: /docs/tasks/configure-pod-container/configure-service-account/
+tags:
+- fundamental
+- core-object
+short-description: >
+  A service account provides an identity for processes that run in a Pod {% glossary_tooltip text="Pods" term_id="pod" %}.
+long-description: >
+  When processes inside Pods access the cluster, they are authenticated by the API server as a particular service account. For example, default. When you create a Pod, if you do not specify a service account, it is automatically assigned the default service account in the same namespace {% glossary_tooltip text="Namespace" term_id="namespace" %}.
```

Comment:
optional: "... processes inside a Pod" (s/Pods/a Pod)
please do change: "authenticated ... particular service account. For example ..." to "authenticated ... particular service account--for example ..." (otherwise unnecessary fragment)
Nice linking to other glossary terms! Nice work all around :-)